openedx/core/djangoapps/theming/helpers.py

```diff
@@ -17,3 +18,35 @@ def get_value(val_name, default=None, **kwargs):
     This is a proxy function to hide microsite_configuration behind comprehensive theming.
     """
     return microsite.get_value(val_name, default=default, **kwargs)
+
+
+def get_template_path(relative_path, **kwargs):
+    """
+    This is a proxy function to hide microsite_configuration behind comprehensive theming.
+    """
+    return microsite.get_template_path(relative_path, **kwargs)
+
+
+def is_request_in_themed_site():
+    """
+    This is a proxy function to hide microsite_configuration behind comprehensive theming.
+    """
+    return microsite.is_request_in_microsite()
+
+
+def get_themed_template_path(relative_path, default_path, **kwargs):
+    """
+    This is a proxy function to hide microsite_configuration behind comprehensive theming.
+
+    It returns path of the themed template (i.e. relative_path) if theming is enabled AND microsite is disabled
+     otherwise it will return path of the microsite's or lms template.
+
+    :param relative_path: relative path of themed template
+    :param default_path: relative path of the microsite's or lms template to use if
+        theming is disabled or microsite is enabled
+    """
+    is_theme_enabled = settings.FEATURES.get("USE_CUSTOM_THEME", False)
```

Comment:
Actually, you can ignore the above remark -- we are going to be consolidating the "USE_CUSTOM_THEME" functionality as part of an upcoming story.